apps/storefront/src/gatsby-theme-docz/components/Sidebar/index.js

```diff
@@ -123,15 +124,18 @@ export const Sidebar = ({ className, open, onClick }) => {
   const routesWithTwoSegments = (doc) =>
     doc.route.replace(/^\/|\/$/g, '').split('/').length < 3
 
-  const subMenus = useMemo(() =>
-    docs
-      .filter(notRootRoute)
-      .filter(routesWithTwoSegments)
-      .map((doc) => ({
-        title: doc.title,
-        route: doc.route,
-        current: current.route.includes(doc.route),
```

Comment:
current is removed, is this intended?

Reply:
Yes, the `<Link />` component handles this on it’s own